extension/jdtls.ext/com.microsoft.gradle.bs.importer/src/com/microsoft/gradle/bs/importer/GradleBuildServerProjectImporter.java

```diff
@@ -61,7 +61,6 @@ public boolean applies(IProgressMonitor monitor) throws OperationCanceledExcepti
             return false;
         }
 
-
```

Comment:
After the change, build server will not support root folder anymore.

At least, false need to be returned here when current workspace contains multiple roots.

Check: `JavaLanguageServerPlugin.getPreferencesManager().getPreferences().getRootPaths()`